website/core/BlogSidebar.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * @providesModule BlogSidebar
+ * @jsx React.DOM
+ */
+
+var MetadataBlog = require('MetadataBlog');
+
+var BlogSidebar = React.createClass({
+  render: function() {
+    return (
+      <div className="nav-docs">
+        <div className="nav-docs-section">
+          <h3>Recent Posts</h3>
+          <ul>
+            {MetadataBlog.files.map(function(post) {
+              return (
+                <li key={post.path}>
+                  <a
+                    className={this.props.title === post.title ? 'active' : ''}
+                    href={'/jest/blog/' + post.path}>
+                    {post.title}
+                  </a>
+                </li>
+              );
+            }.bind(this))}
```

Comment:
I spent 10 minutes getting crazy trying to figure out why it was trying to extract title from undefined :p

for arrow function, I have no idea what transpilers it is running so went the safe way. But yeah, definitely arrow functions are the way to go